go.mod

```diff
@@ -25,3 +25,5 @@ require (
 	golang.org/x/sys v0.0.0-20190712062909-fae7ac547cb7
 	gopkg.in/AlecAivazis/survey.v1 v1.8.5
 )
+
+go 1.13
```

Comment:
@cfergeau why this mention the go version and also this is quite higher version and on RHEL side we still have 1.12.x

Reply:
This is automatically added by go 1.13, and was not intentionally added to the PR, I'll remove it.
https://blog.ksub.org/bytes/2019/09/15/go.mods-go-directive/ has some explanations regarding what this means

Reply:
@cfergeau thanks for sharing, so that means it is just look for the feature set available for a specific version and not stop user to build it on older version as long as it have same feature set. I think it would be better to have 1.12 for now since we are going to use this as part of internal CI and then move to 1.13 once it is updated to RHEL side.

Reply:
Yeah, agreed, I was removing it at the same time I was writing these comments :) Note that the commit changing "expiry" to "expiration" is mostly a RFC as I think both words are ok in the context of certificates.